### PR TITLE
ADD Log traces

### PR DESCRIPTION
**Pull request by dmoranj**
ADD Log traces

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ config.mqtt = {
     /**
      * Host where the MQTT Broker is located.
      */
-    host: 'localhost',
+    host: 'fakedhost',
```
Comment (AlvaroVega) [inline config.js]: Do you really want to change localhost by fakedhost ?

File: config.js [hunk 2]
```diff
@@ -99,32 +99,32 @@ config.iota = {
      * Configuration for the IoT Manager. If the IoT Agent is part of a configuration composed of multiple IoTAgents
      * coordinated by an IoT Manager, this section defines the information that will be used to connect with that manager.
      */
-    iotManager: {
+    //iotManager: {
```
Comment (AlvaroVega) [inline config.js]: Do you really want to remove all iotMaanager conf?

Comment (dmoranj) [inline config.js]: Yes, I do. This is the default config file, and having the IoTManager section uncommented leads to a connection error when the IOTAgent is started, as no one is going to have an IoTManager installed in its own laptop.

Comment (dmoranj) [inline config.js]: NTC

File: config.js (hunk 1 above)
Comment (dmoranj) [inline config.js]: No I don't. Fixed in 	8f2280d

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**